data/membership-schema/src/main/resources/net/corda/schema/membership/member/dynamic/registration/1.0/corda.member.dynamic.registration.json

```diff
@@ -55,14 +55,21 @@
         "O=NotaryService, L=London, C=GB"
       ]
     },
-    "corda.notary.service.plugin": {
-      "description": "The notary plugin type (FQN of notary service plugin class). Valid only when one of the roles is notary.",
+    "corda.notary.service.flow.protocol.name": {
+      "description": "Name of the flow protocol used by the notary service. Valid only when one of the roles is notary.",
       "type": "string",
       "maxLength": 255,
       "examples": [
         "net.corda.notary.MyNotaryService"
       ]
     },
+    "corda.notary.service.flow.protocol.versions": {
+      "description": "List of versions supported for the flow protocol used by the notary service. Valid only when one of the roles is notary.",
+      "type": "array",
+      "items": {
+        "type": "integer"
+      }
```

Comment:
Do we want to enforce a minimum value of 1 for the items (if this can be done)?

Reply:
Added in [ea04b5f](https://github.com/corda/corda-api/pull/955/commits/ea04b5f95b66901793da3d2b018ee03b882e9ffe)